### PR TITLE
Install latest profile if no version is provided

### PR DESCRIPTION
**Pull request by aclevername**
### Description

We will pass "latest" to the catalog API calls if no version is provided. Sister PR https://github.com/weaveworks/profiles/pull/126

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: pkg/catalog/install.go [hunk 1]
```diff
@@ -51,10 +51,10 @@ func Install(cfg InstallConfig) error {
 		},
 		Spec: profilesv1.ProfileSubscriptionSpec{
 			ProfileURL: profile.URL,
-			Version:    filepath.Join(cfg.ProfileName, cfg.Version),
+			Version:    filepath.Join(cfg.ProfileName, profile.Version),
 			ProfileCatalogDescription: &profilesv1.ProfileCatalogDescription{
 				Catalog: cfg.CatalogName,
-				Version: cfg.Version,
+				Version: profile.Version,
 				Profile: cfg.ProfileName,
```
Comment (Callisto13) [inline pkg/catalog/install.go:58]: why not change all these to be `profile.X` i think they all exist there?

doesn't really matter ofc, just wondering why this change is there

**Review by Callisto13 — APPROVED** [review]

File: pkg/catalog/install.go (hunk 1 above)
Comment (aclevername) [reply pkg/catalog/install.go:58]: this is because `profile` contains the actual version, e.g. `v0.1.0`. `cfg.Version` is "latest" when not set. I think your right they could all change to `profile.`

Comment (Callisto13) [reply pkg/catalog/install.go]: ah makes sense
